cumulusci/tasks/tests/test_metadeploy.py

```diff
@@ -145,7 +144,7 @@ def test_run_task(self):
 
         responses.add(
             "GET",
-            "https://metadeploy/products?repo_url=EXISTING_REPO",
+            "https://metadeploy/products?repo_url=https%3A%2F%2Fgithub.com%2FTestOwner%2FTestRepo",
```

Comment:
Lazy fix for test to pass but I'd like to discuss above comment @jstvz

```suggestion
            "https://metadeploy/products?repo_url=https://github.com/TestOwner/TestRepo",
```

Reply:
Fixed in 91a993f